build/webpack.config.js

```diff
@@ -16,12 +16,6 @@ const webpackConfig = {
       stardust: paths.src('index.js'),
     },
   },
-  externals: {
-    bluebird: 'Promise',
-    faker: 'faker',
-    jquery: 'jQuery',
-    lodash: '_',
-  },
```

Comment:
Removed all external CDN scripts, they are now bundled in the doc site vendor bundle.